model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java

```diff
@@ -0,0 +1,34 @@
+package com.ge.predix.acs.model;
+
+import java.util.Set;
+
+public class AdapterResponse {
```

Comment:
`AttributesResponse` will allow for other response types from adapters in future